tox.ini

```diff
@@ -1,8 +1,35 @@
 [tox]
-envlist = py27-django18
+envlist = py{27,34,35,36}-django{18,19,110,111}, flake8
+
+[flake8]
+exclude = *migrations/*
+ignore = F403
```

Comment:
I think we should remove this `ignore` line and use the inline skipping mechanism mentioned in previous comment.

Reply:
agreed